templates/config.edn

```diff
@@ -1,137 +1,167 @@
 {:meta/version 1
 
- ;; Currently, we support either "Markdown" or "Org".
- ;; This can overwrite your global preference so that
- ;; maybe your personal preferred format is Org but you'd
- ;; need to use Markdown for some projects.
- ;; :preferred-format ""
-
- ;; Preferred workflow style.
- ;; Value is either ":now" for NOW/LATER style,
- ;; or ":todo" for TODO/DOING style.
+ ;; Set the preferred format.
+ ;; Available options:
+ ;; - "Markdown" (default)
+ ;; - "Org"
+ :preferred-format "Markdown"
+
+ ;; Set the preferred workflow style.
+ ;; Available options:
+ ;; - ":now" for NOW/LATER style (default)
+ ;; - ":todo" for TODO/DOING style
  :preferred-workflow :now
 
- ;; The app will ignore those directories or files.
- ;; E.g. :hidden ["/archived" "/test.md" "../assets/archived"]
+ ;; Exclude directories/files.
+ ;; Example usage: 
+ ;; :hidden ["/archived" "/test.md" "../assets/archived"]
  :hidden []
 
- ;; When creating the new journal page, the app will use your template if there is one.
- ;; You only need to input your template name here.
+ ;; Define the default journal page template.
+ ;; Enter the template name between the quotes.
  :default-templates
  {:journals ""}
 
- ;; Set a custom date format for journal page title
- ;; Example:
- ;; :journal/page-title-format "EEE, do MMM yyyy"
+ ;; Set a custom date format for the journal page title.
+ ;; Default value: "MMM do, yyyy"
+ :journal/page-title-format "MMM do, yyyy"
```

Comment:
The code uses \`MMM do, yyyy\` as the default if none is specified. I do think that \`EEE, do MMM yyyy\` is better, and I think this option has been set like this for a long time.

https://github.com/logseq/logseq/blob/a5e31128a6366df002488203406684f78d80c7e3/deps/graph-parser/src/logseq/graph_parser/config.cljs#L76-82

@cnrpman, should the default value be updated to \`EEE, do MMM yyyy\`

Reply:
This deviates from the default behavior and is a personal preference.
At least, this should be marked as BREAKING CHANGE.

Reply:
@andelf which is the default behaviour?   
A. `MMM do, yyyy`  
  
B. `EEE, do MMM yyyy`  
  
According to [https://github.com/logseq/logseq/blob/master/deps/graph-parser/src/logseq/graph_parser/config.cljs#L76-82](https://github.com/logseq/logseq/blob/master/deps/graph-parser/src/logseq/graph_parser/config.cljs#L76-82)  
```cljs  
(defn get-date-formatter  
 \[config\]  
 (or  
  (:journal/page-title-format config)  
  ;; for compatibility  
  (:date-formatter config)  
  "MMM do, yyyy"))

```
  
  
the default value is B. Which one should be used as the default?

Reply:
@andelf I appreciate that it's a matter of personal preference, and I'm just curious about which one you would recommend as the default value in the default config template, as well as the expected value if none is provided in the source code.

Reply:
Also btw I just added an example to the docstring.